server/app/views/v1/grid_services/index.json.jbuilder

```diff
@@ -1,3 +1,32 @@
+total_counts = Container.collection.aggregate([
+  { :$match => { grid_id: @grid.id } },
+  { :$group => { _id: "$grid_service_id", total: {:$sum => 1} } }
+])
+running_counts =  Container.collection.aggregate([
+  { :$match => { :grid_id => @grid.id, :'state.running' => true } },
+  { :$group => { _id: "$grid_service_id", total: {:$sum => 1} } }
+])
+
```

Comment:
Could the aggregations rather be part of model? For me it's always a bit weird to run DB operations on view level.